From da92fb02542e0c2a3eed2ceae7c105a0be55065f Mon Sep 17 00:00:00 2001
From: Rene Rebe <rene@exactcode.de>
Date: Sun, 1 Dec 2024 19:13:37 +0100
Subject: [PATCH 63/90] ia64: Use KERN_CONT when printing ITC drift

When printing ITC drift in ia64_init_itm, KERN_CONT was not used,
leading to malformed output.

Add KERN_CONT to the appropriate printk calls.

Signed-off-by: Rene Rebe <rene@exactcode.de>
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/time.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/ia64/kernel/time.c b/arch/ia64/kernel/time.c
index 082101e1b7eb..6cc1569c9325 100644
--- a/arch/ia64/kernel/time.c
+++ b/arch/ia64/kernel/time.c
@@ -309,10 +309,10 @@ void ia64_init_itm(void)
 
 	if (platform_base_drift != -1) {
 		itc_drift = platform_base_drift*itc_ratio.num/itc_ratio.den;
-		printk("+/-%ldppm\n", itc_drift);
+		printk(KERN_CONT "+/-%ldppm\n", itc_drift);
 	} else {
 		itc_drift = -1;
-		printk("\n");
+		printk(KERN_CONT "\n");
 	}
 
 	local_cpu_data->proc_freq = (platform_base_freq*proc_ratio.num)/proc_ratio.den;
-- 
2.25.1

